src/ui/index.js

```diff
@@ -1,15 +1,37 @@
 import 'babel-polyfill';
+import 'isomorphic-fetch';
 import React from 'react';
 import Helmet from 'react-helmet';
 import { render } from 'react-dom';
-import { Router, Route, browserHistory } from 'react-router';
+import { Router, Route, IndexRoute, browserHistory } from 'react-router';
 import injectTapEventPlugin from 'react-tap-event-plugin';
 import { MuiThemeProvider } from 'material-ui';
+import { AccountsClient } from '@accounts/accounts';
+import restClient from '@accounts/rest';
+import '@accounts/react-material-ui';
+import createLogger from 'redux-logger';
+import { accountRoutes, withUser, Authenticated } from '@accounts/react';
 import packageConf from '../../package.json';
 
 injectTapEventPlugin();
 
-const Home = () => <div />;
+AccountsClient.config({
+  server: 'http://localhost:3010',
+  history: browserHistory,
+  title: 'rest-example',
+  loginPath: '/login',
+  signUpPath: '/signup',
+  homePath: '/home',
+  reduxLogger: createLogger(),
+}, restClient);
+
+const Home = withUser(({ user }) =>
```

Comment:
I would love to see support for `user` being an Immutable `Map`, but it's not essential. This is because we are using all Immutable props as much as possible in our app.